spec.bs

```diff
@@ -13,3 +13,540 @@ Group: patcg
+
+  The [=user agent/taxonomy=] and [=user agent/model=] may be shipped to the browser asynchronous w.r.t. the browser release, and may be unavailable at a given point. They must be updated atomically w.r.t. algorithms that access them (e.g. the [=calculate user topics=] algorithm).
+
+  Note: The <a href="https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md">initial taxonomy</a> for Chrome experimentation (i.e. with [=user agent/taxonomy version=] "1") includes hundreds of topics. The taxonomy to be used long term is TBD (<a href="https://github.com/patcg-individual-drafts/topics/issues/3">github issue</a>).
+
+  Each [=user agent=] has an associated topics algorithm configuration (identified by [=browsing topics types/configuration version=] <dfn for="user agent">configuration version</dfn>). The initial value and meaning is browser defined.
+
+  Note: The [=browsing topics types/configuration version=] allows the browser vender to provide algorithms different from the ones specified in this specification. For example, for some of the algorithms in this specification, it may be possible to use a different constant value, while the system overall still has utility and <a href="#meeting-the-privacy-goals-header">meets the privacy goals</a>.
+
+  When [=user agent/configuration version=] is updated, the browser must properly migrate or delete data in [=user agent/user topics state=] and [=user agent/topics history storage=] so that the state and the configuration are consistent.
+
+  <h2 id="browsing-topic-dictionary-header">BrowsingTopic dictionary</h2>
+  The {{BrowsingTopic}} dictionary is used to contain the IDL correspondences of [=browsing topics types/topic id=], [=browsing topics types/version=], [=browsing topics types/configuration version=], [=browsing topics types/taxonomy version=], and [=browsing topics types/model version=].
+
+  <pre class="idl">
+  dictionary BrowsingTopic {
+    [EnforceRange] unsigned long long topic;
+    DOMString version;
+    DOMString configVersion;
+    DOMString modelVersion;
+    DOMString taxonomyVersion;
+  };
+  </pre>
+
+  <div class="example">
+  An example {{BrowsingTopic}} object from Chrome: <code highlight="js">{ configVersion: "chrome.1", modelVersion: "2206021246", taxonomyVersion: "1", topic: 43, version: "chrome.1:1:2206021246" }</code>.
+  </div>
+
+  <div algorithm>
+    A {{BrowsingTopic}} dictionary |a| is <dfn id=browsing-topics-dictionary-less-than-comparator for="browsing-topic">code unit less than</dfn> a {{BrowsingTopic}} dictionary |b| if the following steps return true:
+    1. If |a|["{{BrowsingTopic/topic}}"] < |b|["{{BrowsingTopic/topic}}"], then return true.
```

Comment:
```suggestion
    1. If |a|["{{BrowsingTopic/topic}}"] is [=/code unit less than=] |b|["{{BrowsingTopic/topic}}"], then return true.
```

Reply:
The {{BrowsingTopic/topic}} isn't a string. Can we still say [=/code unit less than=]?

Reply:
You're totally right, this is my bad. Please ignore.